### DISCUSSION TITLE
Significant digits in observations data

### DISCUSSION THREAD

**Discussion by sthiboldeaux** [weather-gov/api#645] [Q&A]
Hi there, 
I am using the following url to return the most recent observations from the NWS station in Caribou, Maine: 
https://api.weather.gov/stations/KCAR/observations/

I am pulling the hourly precip. data from this response, but it appears that the values are rounded to the nearest centimeter (the only observations that have values other that 'None' are 10 mm, 20 mm, etc.). 

The rounding appears to be correct when I look at the NWS station website (below), but I would like to precision to be the same as shown on the timeseries viewer (0.00 mm). 
https://www.weather.gov/wrh/timeseries?site=KCAR 

Is there a way to modify the unit precision in the url? 

Thanks,
Sean

**Comment by LorenzoPeve** [comment]
This issue still exists @nickmachairas.
